Ksenia Myakinkaya/js-1.js

```diff
@@ -0,0 +1,103 @@
+function splitAndMerge(str, sp){
+    let words=str.split(" ");
+    let newWords=[];
+    words.forEach(elem => {
+        let characters=elem.split("")
+        newWords.push(characters.join(sp))
+    });
+    return newWords.join(" ");
+}
+
+function convert(obj){
+    let arr=[];
+    for (elem in obj){
+        arr.push([elem,obj[elem]])
+    }
+    return arr
+}
+
+function toCamelCase(str){
+    let arr=str.split(/-|_/)
+    let camelCaseStr=arr[0]
+    for(let i=1; i<=arr.length-1; i++){
+        camelCaseStr+=arr[i][0].toUpperCase()+arr[i].substr(1)
+    }
+    return camelCaseStr
+}
+
+function reverseWords(str){

```

Comment:
Output should be " A nuf elttil !egnellahc " instead of "A nuf elttil !egnellahc"